src/controllers/AbstractRestController.ts

```diff
@@ -0,0 +1,29 @@
+import { inject } from 'inversify';
+import { BaseHttpController, httpGet, controller } from 'inversify-express-utils';
+import { RedisService } from '../services/RedisService';
+import { TYPES } from '../InjectableTypes';
+
+@controller('/')
+export abstract class AbstractRestController extends BaseHttpController {
+
+    constructor(@inject(TYPES.RedisService) protected readonly redisService: RedisService) {
+        super();
+    }
+
+    @httpGet('healthCheck')
+    public async healthCheck(): Promise<void> {
+        await this.sendInfo();
+    }
+
+    private async sendInfo(): Promise<void> {
+        const obj: any = await this.redisService.checkHealth();
+        this.httpContext.response.send(
+            `
+            <p>Redis online:  ${this.redisService.ping()}</p>
+            <p>Number of Times Visited: ${obj.n}\n</p>
```

Comment:
Is it relevant know the no. times visited and last time visited? Do we just want to know the connection status?

Reply:
It shows evidence that we can retrieve and store data from redis which is quite important since we don't have a testing framework yet

Reply:
ok, that's fair enough for the time being